armi/physics/neutronics/globalFlux/globalFluxInterface.py

```diff
@@ -46,7 +47,7 @@ class GlobalFluxInterface(interfaces.Interface):
     Should be subclassed by more specific implementations.
     """
 
-    name = None  # make sure to set this in subclasses
+    name = "GlobalFlux"  # make sure to set this in subclasses
```

Comment:
Is the comment `make sure to set this in subclasses` still relevent?

Reply:
Also, this should probably be `function`, rather than name

Reply:
* Yes it has to be set in subclasses to get the proper name in file labels and printouts
* No it shouldn't be function because it needs to change for different particular implementations of the `globalFlux` function